spec.html

```diff
@@ -26011,6 +26858,137 @@ <h1>Properties of Number Instances</h1>
+        <li>is the initial value of the `BigInt` property of the global object.</li>
+        <li>performs a type conversion when called as a function rather than as a constructor.</li>
+        <li>is not intended to be used with the `new` operator or to be subclassed. It may be used as the value of an `extends` clause of a class definition but a `super` call to the `BigInt` constructor will cause an exception.</li>
+      </ul>
+
+      <emu-clause id="sec-bigint-constructor-number-value">
+        <h1>BigInt ( _value_ )</h1>
+        <p>When `BigInt` is called with argument _value_, the following steps are taken:</p>
+        <emu-alg>
+          1. If NewTarget is not *undefined*, throw a *TypeError* exception.
+          1. Let _prim_ be ? ToPrimitive(_value_, hint Number).
+          1. If Type(_prim_) is Number, return ? NumberToBigInt(_prim_).
+          1. Otherwise, return ? ToBigInt(_value_).
+        </emu-alg>
+
+        <emu-clause id="sec-number-to-bigint" aoid="NumberToBigInt">
+          <h1>Runtime Semantics: NumberToBigInt ( _number_ )</h1>
+          <emu-alg>
+            1. Assert: Type(_number_) is Number.
+            1. If IsInteger(_number_) is *false*, throw a *RangeError* exception.
+            1. Return a BigInt representing the mathematical value of _number_.
+          </emu-alg>
+        </emu-clause>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-bigint-constructor">
+      <h1>Properties of the BigInt Constructor</h1>
+      <p>The value of the [[Prototype]] internal slot of the BigInt constructor is the intrinsic object %FunctionPrototype%.</p>
+      <p>The BigInt constructor has the following properties:</p>
+
+      <emu-clause id="sec-bigint.asintn">
+        <h1>BigInt.asIntN ( _bits_, _bigint_ )</h1>
+        <p>When the `BigInt.asIntN` is called with two arguments _bits_ and _bigint_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _bits_ be ? ToIndex(_bits_).
+          1. Let _bigint_ be ? ToBigInt(_bigint_).
+          1. Let _mod_ be a BigInt representing _bigint_ modulo 2<sup>_bits_</sup>.
+          1. If _mod_ &ge; 2<sup>_bits_ - 1</sup>, return _mod_ - 2<sup>_bits_</sup>; otherwise, return _mod_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-bigint.asuintn">
+        <h1>BigInt.asUintN ( _bits_, _bigint_ )</h1>
+        <p>When the `BigInt.asUintN` function is called with two arguments _bits_ and _bigint_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _bits_ be ? ToIndex(_bits_).
+          1. Let _bigint_ be ? ToBigInt(_bigint_).
+          1. Return a BigInt representing _bigint_ modulo 2<sup>_bits_</sup>.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-bigint.prototype">
+        <h1>BigInt.prototype</h1>
+        <p>The initial value of `BigInt.prototype` is the intrinsic object %BigIntPrototype%.</p>
```

Comment:
now that we have the new intrinsics format, let's just remove this intrinsic entirely.
```suggestion
```